src/discordtipbot/discordbot.cr

```diff
@@ -711,21 +713,37 @@ class DiscordBot
 
     return reply(msg, "Current total user balances: **#{@tip.db_balance}**") if cmd.size == 1
 
-    if cmd[1] == "unclaimed"
+    case cmd[1]?
+    when "unclaimed"
       node = @tip.node_balance
       return if node.nil?
       unclaimed = node - (@tip.deposit_sum - @tip.withdrawal_sum)
 
       return reply(msg, "Unclaimed coins: **#{unclaimed}** #{@config.coinname_short}")
-    end
-
-    if cmd[1]? == "balance"
+    when "balance"
       return reply(msg, "**ERROR**: You forgot to supply an ID to check balance of") unless cmd[2]?
       bal = @tip.get_balance(cmd[2].to_u64)
       reply(msg, "**#{cmd[2]}**'s balance is: **#{bal}** #{@config.coinname_short}")
     end
   end
 
+  def admin_config(msg : Discord::Message, cmd_string : String)
+    return if admin_alarm?(msg)
+
+    # Currently the same coins have to be present in both the old and new config file
+
+    # cmd[0] = command, cmd [1] = path
+    cmd = cmd_string.split(" ")
+
+    begin
+      Config.reload(cmd[1]?)
+    rescue
+      return reply(msg, "There was an issue loading config from `#{cmd[1]?}`")
```

Comment:
Maybe add `rescue ex`, then `raise ex` after replying so the error is logged and you know exactly what happened? Or include `ex` as part of the `reply`?